src/order/complete_lattice.lean

```diff
@@ -416,6 +416,10 @@ protected lemma function.surjective.supr_congr {g : ι' → α} (h : ι → ι')
   (h2 : ∀ x, g (h x) = f x) : (⨆ x, f x) = ⨆ y, g y :=
 by { convert h1.supr_comp g, exact (funext h2).symm }
 
+protected lemma equiv.supr_congr {g : ι' → α} (e : ι ≃ ι') (h : ∀ x, g (e x) = f x) :
+  (⨆ x, f x) = ⨆ y, g y :=
+e.surjective.supr_congr _ h
```

Comment:
Can you add the `supr_comp` version too to match `function.surjective.supr_comp`?